calblueprint/meepanyar
# To Collect and To Meter go to appropriate tabs

[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR
The "To Meter" and "To Collect" buttons now go to the appropriate tabs in `CustomerMain`

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## How to review
1. Click on the "To Meter" button and make sure it goes to the right tab
2. Click on the "To Collect" button and make sure it goes to the right tab
3. Click on the Customer icon in the bottom navbar and make sure it doesn't break (it should go to ALL)

[//]: # 'The order in which to review files and what to expect when testing locally'

## Relevant Links

### Online sources

[//]: # 'Optional - copy links to any tutorial or documentation that was useful to you when working on this PR'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## Next steps

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

### Screenshots

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @julianrkung

[//]: # 'This tags Julian as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
